.travis.yml

```diff
@@ -4,6 +4,12 @@ language: java
 
 cache:
 
+dist: trusty
```

Comment:
Trusty is the name of used Ubuntu distribution. Without a named distribution the default from TravisCI is used and this is at the moment Precise. Precise is since May 4th deprecated in TravisCI and does not support Java8. So I switched explicit to next distribution to get current build processes run.